user_guide/user_guide.md

```diff
@@ -0,0 +1,118 @@
+# User Guide
+
+## Register User
+Before you can start with the MTPP you have to register your own user.
+
+*Click on the Button "You're new to MTPP? Sign Up!".*
+
+![Login Page](/resources/images/login_page.png)
```

Comment:
you could not know that, but today I found out, that the absolute path /resources/... does work for the .md files, but the HTML files (for github pages) cant access those directly.. it would be great if you could change all the URLs here to ../resources/ (I already applied this to all our other sources)